src/amber/server/server.cr

```diff
@@ -61,23 +61,22 @@ module Amber
       Signal::INT.trap do
         logger.info "Shutting down Amber"
         server.close
-        exit
       end
```

Comment:
@c910335 I think the `exit` is needed here otherwise we won't be able to exit amber process inside the signal trap.

Perhaps, we can use `at_exit` instead:

```cr
at_exit do
  logger.info "Shutting down Amber"
  server.close
end
```

WDYT?

Reply:
`Signal::INT.trap` is for ^c, and `at_exit` is for `exit`.
Their purposes are different.

Reply:
I think `server.close` is enough to exit the process.

Reply:
I found that adding `Signal::INT.reset` would be better.

```crystal
Signal::INT.trap do
  Signal::INT.reset
  logger.info "Shutting down Amber"
  server.close
end
```

Then the first ^c will try to close the server, and the further ^c will kill the process.

WDYT?

Reply:
@c910335 Interesting idea, let me try that :+1:

Reply:
@c910335 Can you add `Signal::INT.reset`? so I can approve this :sweat_smile:

Reply:
@faustinoaq done